ios/RNSentryTimeToDisplay.m

```diff
@@ -0,0 +1,43 @@
+#import "RNSentryTimeToDisplay.h"
+#import <QuartzCore/QuartzCore.h>
+#import <React/RCTLog.h>
+
+@implementation RNSentryTimeToDisplay
```

Comment:
Q: I wonder if we could reuse the implementation of [`SentryDisplayLinkWrapper`](https://github.com/getsentry/sentry-cocoa/blob/main/Sources/Sentry/include/SentryDisplayLinkWrapper.m) on this class.

Reply:
SentryDisplayLinkWrapper is not available for RN, we could do it, but I think this Is good enough.

Reply:
Makes sense 👍 Thank you for your response 🙇